README.md

```diff
@@ -26,7 +26,7 @@ use Red:api<2>;
 model Person {...}
 
 model Post is rw {
-    has Int         $.id        is serial;
+    has Int         $.id        is serial; # an id field is mandatory
```

Comment:
`id` is not mandatory. Only if you want to be able to update

Reply:
I've eliminated one and clarified the other. Both have an id field, so it's maybe convenient to say why it's there.